src/internal/m365/collection/drive/url_cache_test.go

```diff
@@ -509,63 +449,3 @@ func (suite *URLCacheUnitSuite) TestNeedsRefresh() {
 	cache.lastRefreshTime = time.Now()
 	require.False(t, cache.needsRefresh())
 }
-
-// Test newURLCache
-func (suite *URLCacheUnitSuite) TestNewURLCache() {
```

Comment:
we should probably keep these tests. They check important edge cases that would be good to have "documented" somewhere so we know the behavior doesn't change

Reply:
agreed and will do.  Can't remember why I originally took them out.